vmdb/app/controllers/catalog_controller.rb

```diff
@@ -205,7 +205,8 @@ def explorer
       @built_trees << build_st_tree
       @accords.push(:name => "sandt", :title => "Catalog Items", :container => "sandt_tree_div")
     end
-    if role_allows(:feature => "orchestration_templates_accord")
+    if role_allows(:feature => "orchestration_templates_view") ||
```

Comment:
@mzazrivec you can do role_allows(:feature => "orchestration_templates_accord", :any => true) here instead of checking for features separately, role_allows returns true if any of the child features are checked under a feature.